scripts/compare-artifacts.ts

```diff
@@ -0,0 +1,38 @@
+import yargs from "yargs";
```

Comment:
Isn't this included in PR #80?

Reply:
Thanks for this, I have no idea how it got there :-)